pyro/contrib/gp/models/model.py

```diff
@@ -60,7 +58,7 @@ class GPModel(Parameterized):
       <http://pyro.ai/examples/svi_part_i.html>`_:
 
         >>> optimizer = pyro.optim.Adam({"lr": 0.01})
-        >>> svi = SVI(gpr.model, gpr.guide, optimizer, loss=Trace_ELBO())
+        >>> svi = pyro.infer.SVI(gpr.model, gpr.guide, optimizer, loss=pyro.infer.Trace_ELBO())
```

Comment:
`SVI`, and `Trace_ELBO` should already be available in the doctest namespace, so you don't need to use absolute path. If you are adding it to be explicit, that's fine though!

Reply:
I also think so, but triggered errors for doctest at this place. :( I'll test it again.

Reply:
Try running `make doctest` for the correct command.

Reply:
I can't run `make doctest` in my local machine because of the error:`ImportError: Error importing plugin "tests.doctest_fixtures": No module named 'tests.doctest_fixtures'`. In last commit, I remove `pyro.infer` and this error is triggered in Travis: `UNEXPECTED EXCEPTION: NameError("name 'SVI' is not defined",)` at these places

Reply:
@neerajprad I can import `tests.common` and can do other `make foo` except doctest. ^^! I have installed all required pytest plugins and used `pytest=3.7.4` as suggested in `setup.py` file. I installed most of my packages using conda instead of `pip`. That might be the problem.